### PR TITLE
Isfulladmin py

### PR DESCRIPTION
**Pull request by will-moore**
# What this PR does

Fixes test from #5483 (replaces that PR).

# Testing this PR

1. Check test is green

cc @dominikl

### CODE REVIEW

File: components/tools/OmeroPy/src/omero/gateway/__init__.py [hunk 1]
```diff
@@ -2324,6 +2324,19 @@ def isAdmin(self):
...
 
+    def isFullAdmin(self):
+        """
+        Checks if a user has full administration privileges.
+
+        :return:    Boolean
+        """
+
+        if self.getEventContext().isAdmin:
+            allPrivs = list(self.getEnumerationEntries('AdminPrivilege'))
```
Comment (mtbc) [inline components/tools/OmeroPy/src/omero/gateway/__init__.py:2335]: Would `allPrivs` be worth caching, or noting upon connection? (Oh, or does `getEnumerationEntries` already do that?)

**Comment by dominikl** [comment]
👍 Thanks @will-moore !

File: components/tools/OmeroPy/src/omero/gateway/__init__.py (hunk 1 above)
Comment (will-moore) [reply components/tools/OmeroPy/src/omero/gateway/__init__.py:2335]: I guess it depends how expensive it is to ```getEnumerationEntries()```. I don't imagine you'd be calling this multiple times in normal usage?

Comment (mtbc) [reply components/tools/OmeroPy/src/omero/gateway/__init__.py:2335]: Fair enough, not much more expensive than the server roundtrip overhead. Not a blocker.

**Comment by joshmoore** [comment]
Closing and re-opening now that https://github.com/openmicroscopy/openmicroscopy/pull/5498 is merged

**Comment by pwalczysko** [comment]
See https://github.com/will-moore/openmicroscopy/pull/58#issuecomment-328887423 - possible broken tests here.

**Comment by will-moore** [comment]
https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-Python27/656/testReport/OmeroPy.test.integration.clitest.test_ldap/TestLDAP/testAdminOnly_list_/
failing...

**Comment by pwalczysko** [comment]
The commit https://github.com/openmicroscopy/openmicroscopy/pull/5494/commits/504625cf42015d626b03733b419a6a4413ee4c0e had the desired effect, the tests are passing now. https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-python/lastCompletedBuild/testReport/OmeroPy.test.integration.clitest.test_ldap/TestLDAP/ and
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-Python27/lastCompletedBuild/testReport/OmeroPy.test.integration.clitest.test_ldap/TestLDAP/

Happy with the change as well, although the test was fully removed for the ``list`` subcommand, it was never doing anything else but checking that it is ``AdminOnly`` in the first place, so no empoverishment of the test suite happened.

**Comment by will-moore** [comment]
@joshmoore was going to have a final check of this...?

**Comment by joshmoore** [comment]
https://github.com/openmicroscopy/openmicroscopy/blob/v5.3.4/components/server/src/ome/logic/AdminImpl.java#L330 (backed by https://github.com/openmicroscopy/openmicroscopy/blob/v5.3.4/components/server/src/ome/logic/LdapImpl.java#L657) does allow regular users to list others' distinguised name, but this seems fine to me. :+1:

**Pull request merged**